_pages/venue.md

```diff
@@ -5,32 +5,24 @@ subtitle: Venue
 permalink: /venue/
 ---
 
-StarCon was held in the [Mathematics 3](https://www.google.com/maps/place/Mathematics+3+(M3)/@43.473224,-80.5441028,15z/data=!4m5!3m4!1s0x0:0x2ee32a10cd3a1d73!8m2!3d43.473224!4d-80.5441028) building at the University of Waterloo.
+StarCon will be held in the [University of Waterloo School of Optometry and Vision Science](https://www.google.com/maps/place/School+of+Optometry+and+Vision+Science/@43.4755575,-80.5455983,15z/data=!4m2!3m1!1s0x0:0xb34254f15d15b6b1?sa=X&ved=2ahUKEwjKusuX14zeAhVKFzQIHcRBBWQQ_BIwEXoECAcQCw) (OPT).
 
 ## WiFi
 
-Information on how to connect to wifi is [here](/wifi).
+There will be WiFi provided. Information on how to connect to wifi will be updated [here](/wifi) closer to the event.
 
 ## Amenities
 
-There will be a quiet room (M3 3127) available for those that need a break from the conference at any time.
+There will be a quiet room (room TBA) available for those that need a break from the conference at any time.
 
-There are unfortunately no gender neutral restrooms in M3 but there are [some nearby](https://uwaterloo.ca/equity/equity-initiatives/inclusive-washrooms/inclusive-washroom-list) (the closest are 4 restrooms in the Davis Center).
+There are unfortunately no gender neutral restrooms in OPT but there are [some nearby](https://uwaterloo.ca/equity/equity-initiatives/inclusive-washrooms/inclusive-washroom-list).
```

Comment:
we're gonna make the bathrooms gender neutral (verify in #logistics)

Reply:
but are we allowed to? is it bad to officially write it here?

Reply:
--> #logistics

Reply:
```suggestion
Gender neutral bathrooms will be available.
```